src/DonorProfiles/resources/js/app/components/donation-receipt/index.js

```diff
@@ -44,7 +44,9 @@ const DonationReceipt = ( { donation } ) => {
 							<FontAwesomeIcon icon="globe" /> { __( 'Address:', 'give' ) }
 						</div>
 						<div className="give-donor-profile-donation-receipt__value">
-							{ donor.address }
+							{ donor.address.street } <br />
```

Comment:
@henryholtgeerts can we make address formatting dynamic because an extra line will be added if any entry is missing.

Reply:
Good thinking! Done, ref: https://github.com/impress-org/givewp/pull/5566/commits/5cccf424904c6d3a39f2a372257eb2584ecc8f4c